.gitignore

```diff
@@ -6,16 +6,19 @@
 /configure
 /libtool
 /src/config.h.in
+/sysinfo.gap-*
 
 /gen/
 /obj/
 
-/build/
```

Comment:
This line was in there intentionally, to support my out-of-tree build setup. Please leave it in.